From bf3fe491b0dbaab63d393da3d00ddab40a105a76 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Tue, 24 Sep 2013 09:45:11 +0200
Subject: [PATCH 566/628] usb: gadget: Fix ch9_cmdbuf structure

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 9cf01f02ef4282a47ccebdbf4526a501ff6df49d

- Fix kernel-doc format
- Define structure and then ch9_cmdbuf variable

Acked-by: Subbaraya Sundeep Bhatta <sbhatta@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/usb/gadget/xilinx_udc.c |   10 ++++++----
 1 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/drivers/usb/gadget/xilinx_udc.c b/drivers/usb/gadget/xilinx_udc.c
index ac22841..634038f 100644
--- a/drivers/usb/gadget/xilinx_udc.c
+++ b/drivers/usb/gadget/xilinx_udc.c
@@ -206,8 +206,8 @@ struct xusb_udc {
 /* Global xusb_udc variable*/
 static struct xusb_udc controller;
 
-/*
- * ch9_cmdbuf - Standard USB Command Buffer Structure defined
+/**
+ * struct cmdbuf - Standard USB Command Buffer Structure defined
  * @setup: usb_ctrlrequest structure for control requests
  * @contreadptr: pointer to endpoint0 read data
  * @contwriteptr: pointer to endpoint0 write data
@@ -217,7 +217,7 @@ static struct xusb_udc controller;
  * @setupseqrx: rx status
  * @contreaddatabuffer: read data buffer for endpoint0
  */
-static struct {
+struct cmdbuf {
 	struct usb_ctrlrequest setup;
 	u8 *contreadptr;
 	u8 *contwriteptr;
@@ -226,7 +226,9 @@ static struct {
 	u32 setupseqtx;
 	u32 setupseqrx;
 	u8 contreaddatabuffer[64];
-} ch9_cmdbuf;
+};
+
+static struct cmdbuf ch9_cmdbuf;
 
 /*
  * Initial Fixed locations provided for endpoint memory address
-- 
1.7.5.4

